rb/lib/selenium/webdriver/common/driver.rb

```diff
@@ -318,7 +318,8 @@ def ref
       attr_reader :bridge
 
       def create_bridge(caps:, url:, http_client: nil)
-        Remote::Bridge.new(http_client: http_client, url: url).tap do |bridge|
+        klass = caps['webSocketUrl'] ? Remote::BiDiBridge : Remote::Bridge
```

Comment:
Oh, hai handshake bridge :)

Reply:
No! We're doing this before the session start call! It isn't based on what is coming back from the driver. 😂

Reply:
Ah, I thought we do it by checking for webSocketUrl in response capabilities.

Reply:
That was the previous PR; it extended a module based on the response. But (shocking) it added complexity in other places and I realized we don't actually need to do that!